tests/unit/test_log.py

```
@@ -31,13 +32,18 @@ def setUp(self):
+            f"s3_upload.log.{datetime.now().strftime('%Y-%m-%d')}",
+        )
+
+        log.set_file_handler(self.logger, TEST_DATA_DIR)
         self.logger.setLevel(5)
 
     def tearDown(self):
-        log_file = os.path.join(Path(__file__).parent, "s3_upload.log")
-        if os.path.exists(log_file):
-            os.remove(log_file)
+        log.clear_old_logs(
+            logger=self.logger, log_dir=TEST_DATA_DIR, backup_count=-1
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)